docs/public/deploying-a-devfile-using-odo.adoc

```diff
@@ -278,28 +313,29 @@ In this example, we will be deploying the same Java Spring Boot® component we d
 ----
 +
 
-. Create a component configuration using the `java-spring-boot` component-type named `mydockerspringboot`:
+. Create a component configuration using the `java-spring-boot` component-type named `mydockerspringboot` and download its sample project:
 +
 [source,sh]
 ----
-  $ odo create java-spring-boot mydockerspringboot
+  $ odo create java-spring-boot --downloadSource
```

Comment:
See above comment regarding why we shouldn't add `--downloadSource` or at least put it on a note.